extensions/email/src/main/java/org/mule/extension/email/api/sender/SMTPConfiguration.java

```diff
@@ -32,10 +41,32 @@
     private String from;
 
     /**
+     * Default encoding to be used if there is encoding specified in {@link EmailContent#charset}
```

Comment:
jdoc is wrong. This is used if NO enconding is specified. Also rephrase this to not reference one particular entity. Make it general since it more that surely will end up applying to some other stuff